homeassistant/components/sector_alarm/alarm_control_panel.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from homeassistant.components.sector_alarm import DOMAIN as SECTOR_DOMAIN
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.const import (STATE_ALARM_DISARMED, STATE_ALARM_ARMED_HOME,
+                                 STATE_ALARM_ARMED_AWAY, STATE_ALARM_PENDING)
+
+import homeassistant.components.sector_alarm as sector_alarm
+
+DEPENDENCIES = ['sector_alarm']
+
+DOMAIN = 'sector_alarm'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Initial setup of the platform."""
+    sector_connection = hass.data.get(SECTOR_DOMAIN)
+    code = discovery_info[sector_alarm.CONF_CODE]
+    panel_id = discovery_info[sector_alarm.CONF_ALARM_ID]
+
+    async_add_entities([SectorAlarmPanel(sector_connection, panel_id, code)])
+
+
+class SectorAlarmPanel(alarm.AlarmControlPanel):
+    """Get the alarm status, and arm/disarm alarm."""
+
+    def __init__(self, sectorConnect, panelId, code):
+        self._panelid = panelId
+        self._code = code
+        self._sectorconnect = sectorConnect
```

Comment:
You are being inconsistent in naming parameters between here and the rest of the code